packages/localizations/src/en-US.ts

```diff
@@ -898,4 +902,18 @@ export const enUS: LocalizationResource = {
       web3WalletButtonsBlockButton: '{{provider|titleize}}',
     },
   },
+  waitlist: {
```

Comment:
Do we need to run the localization command to add `undefined` for other locales?